redaxo/src/core/lib/console/user/create.php

```diff
@@ -73,6 +73,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $user->setValue('login', $login);
         $user->setValue('password', rex_backend_login::passwordHash($password));
         $user->setValue('admin', $input->getOption('admin') ? 1 : 0);
+        $user->setValue('login_tries', 0);
```

Comment:
macht es eigentlich mehr sinn es in dem table schema festzuhalten?
https://github.com/redaxo/redaxo/blob/0f61c0e4338f26b8ca31fd31c6bdc1515e7b3afd/redaxo/src/core/install.php#L39

Reply:
stimmt, wäre auch gut. kannst du den PR ändern?